stacker/util.py

```diff
@@ -366,3 +369,88 @@ def read_value_from_path(value):
         with open(relative_path) as read_file:
             value = read_file.read()
     return value
+
+
+def sanitize_git_path(uri, ref='HEAD'):
```

Comment:
I'd change `ref` to `ref=None`, and then in the function do:

`ref = ref or "HEAD"` to set the default